api/v1alpha1/sleepinfo_types.go

```diff
@@ -144,6 +163,10 @@ func (s SleepInfo) GetWakeUpSchedule() (string, error) {
 	return s.getScheduleFromWeekdayAndTime(s.Spec.WakeUpTime)
 }
 
+func (s SleepInfo) GetIncludeRef() []IncludeRef {
```

Comment:
Can you add a test on this function? As done for the `GetExcludeRef` one

Reply:
I have added tests for this as well